.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,19 @@
+---
+# This workflow sets the test / all status check to success in case it's a docs only PR and test.yml is not triggered
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+name: test
+
+on:
+  pull_request:
+    paths:
```

Comment:
Will this particular workflow run even if changes contain other files that are not `.md` or `.asciidoc`? 

I've seen this pattern with `paths-ignore`:
```
    paths-ignore: # This expression needs to match the paths ignored on ci.yml.
      - '**' # Ignore all file changes, except for the below.
      - '!**.md'
      - '!**.asciidoc'
      - ```

Reply:
Ah, yes. Thank you for the hint. I was following the github docs and there it's kind of reversed.. which is why it works in their usecase.

After thinking about it.. your suggestion is necessary, so it won't trigger in case other files are edited as well.

Reply:
https://github.com/elastic/apm-agent-python/pull/1753/commits/33867e5ca2f606c6e6c672bc02808a40f2e79247